public/javascripts/ng-controllers.js

```diff
@@ -40,7 +40,30 @@ springInitiative.controller('loginCtrl', function($scope, $http, $location) {
   }
 });
 
-springInitiative.controller('mainController', function($scope, $http) {
+
+
+springInitiative.controller('indexController', function($scope, $rootScope, $http, $location) {
+  $scope.students = [];
+
+  $scope.user = $rootScope.loggedInUser;
+  console.log($rootScope.loggedInUser);
+
+  $http.get('/api/allStudents').then(function(data) {
+    console.log(data);
+    $scope.students = data.data;
+  }, function(err) {
+    console.log('Error: in GET \'/student\'', err);
+  });
+});
+
+springInitiative.controller('overviewController', function($scope, $rootScope, $http, $location) {});
+
+springInitiative.controller('programController', function($scope, $rootScope, $http, $location) {
+  $scope.programName = 'something spring';
+  $scope.programInfo = ':3';
+});
```

Comment:
Yes. I think Nora is fixing it in a sub-branch

On Sat, Apr 2, 2016 at 5:37 PM Shruti Iyer <notifications@github.com> wrote:

> In public/javascripts/ng-controllers.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/25#discussion_r58300286>
> :
>
> > +  console.log($rootScope.loggedInUser);
> > +
> > +  $http.get('/api/allStudents').then(function(data) {
> > +    console.log(data);
> > +    $scope.students = data.data;
> > +  }, function(err) {
> > +    console.log('Error: in GET \'/student\'', err);
> > +  });
> > +});
> > +
> > +springInitiative.controller('overviewController', function($scope, $rootScope, $http, $location) {});
> > +
> > +springInitiative.controller('programController', function($scope, $rootScope, $http, $location) {
> > +  $scope.programName = 'something spring';
> > +  $scope.programInfo = ':3';
> > +});
>
> Are the above lines just placeholders for now?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/25/files/e3296356c8a1c2a4604e115df8a4f0c74fd406a4#r58300286>
>